tests/sles4sap/trento/deploy_trento.pm

```diff
@@ -37,14 +38,48 @@ sub run {
     upload_logs($deploy_script_log);
 
     my $trento_registry_chart = get_var(TRENTO_REGISTRY_CHART => 'registry.suse.com/trento/trento-server');
+    my $cfg_json = 'config_images_gen.json';
+    my @imgs = qw(WEB RUNNER);
+    if (get_var("TRENTO_REGISTRY_IMAGE_$imgs[0]") || get_var("TRENTO_REGISTRY_IMAGE_$imgs[1]")) {
```

Comment:
I don't understand the need for this if conditional here, when you check for those settings again on L51.

Reply:
There are multiple TRENTO_REGISTRY_IMAGE_*** variables, all the code block from L43 is executed if at least one of them exist, al the other tests are dedicated to one particular TRENTO_REGISTRY_IMAGE_*** 

I can do something like
```
my $unreleased_trento = get_var("TRENTO_REGISTRY_IMAGE_$imgs[0]") || get_var("TRENTO_REGISTRY_IMAGE_$imgs[1]");

if ($unreleased_trento) {
```

Reply:
Disregard my comment. I see now that you have some command executions between L62-L64